pyaerocom/aeroval/modelmaps_engine.py

```diff
@@ -126,10 +132,16 @@ def _process_map_var(self, model_name, var, reanalyse_existing):
             If the data has the incorrect number of dimensions or misses either
             of time, latitude or longitude dimension.
         """
-        data = self.read_model_data(model_name, var)
+
+        try:
+            data = self.read_model_data(model_name, var)
+        except Exception as e:
+            logger.warning(f"{e}")
```

Comment:
Longer message: `f"Skipping due to exception during reading of model {model_name} ({var}): {e}"`

And very strange: why do you catch the Exception here, and then you catch most exceptions in line 94. I would solve that with a special exception, i.e. `raise ModelUnavailableException` which could be caught in line 94. Exceptions are nothing else than special return statements, much better than a dummy, unspecified `return None` in line 140. (Is None what causes the TypeError in line 91?)

Reply:
https://github.com/metno/pyaerocom/blob/main-dev/pyaerocom/exceptions.py has already a VariableNotFoundError, which might be applicable here? Or you define a new exception.